### ISSUE TITLE
[hud not rendering] <hud not rendering>

### ISSUE THREAD

**Issue by DevEriJMT** [PojavLauncherTeam/PojavLauncher#4596]
### Describe the bug

in versions 1.17 onwards, every time I enter the hud it simply does not render, yes with optifine it can be solved by using antialiasing or adding shaders but performance is lost by not using sodium and it is somewhat annoying not being able to use certain mods because they are not compatible with optifabric, I wish they could fix that.

### The log file and images/videos

https://github.com/PojavLauncherTeam/PojavLauncher/assets/140129917/c6943398-2637-45de-aa59-db01d14b8c4b

https://github.com/PojavLauncherTeam/PojavLauncher/assets/140129917/720f6c78-785d-4793-84be-d184f1ed7daf

[latestlog.txt](https://github.com/PojavLauncherTeam/PojavLauncher/files/12644097/latestlog.txt)

### Steps To Reproduce

```markdown
I start a version that is 1.17 or higher than this
```

### Expected Behavior

I didn't understand the question or the example

### Platform

```markdown
- Device model: Lenovo TB-X306X (LenovoTB-X306)
- CPU architecture: aarch64
- Android version: 10
- PojavLauncher version: dahlia-740-623f7dd33-v3_openjdk(latest of playstore)
```

### Anything else?

None

**Comment by GaucovyAero** [comment]
## Workaround: graphical issues on the PowerVR GE8320 on Holy GL4ES 1.1.5, 1.17+

1. Install the latest GitHub Actions build of PojavLauncher here: https://github.com/PojavLauncherTeam/PojavLauncher/actions
(click on the latest `v3_openjdk` build with a green checkmark on it, log into your GitHub account, download app-debug, extract app-debug.zip, install app-debug.apk).
2. Create a Quilt profile and put `-Dloader.disable_forked_guis=true` into your Quilt profile's JVM arguments (edit the profile with the pen button)
3. Install Quilted Fabric API, Cloth Config, Mod Menu, Methane and Axolotl Client
4. Launch your game
5. In the main menu (Minecraft), click on Mods -> AxolotlClient -> Settings button (in the top right corner) -> Client Options -> Rendering -> Hit Color (scroll down), and change Alpha to 0.

**Comment by DevEriJMT** [comment]
Hello, I don't know English, I'm using a translator and I didn't understand anything lol

**Comment by DevEriJMT** [comment]
Hello, I don't know English, I'm using a translator and I didn't even understand anything lol

**Comment by GaucovyAero** [comment]
> Hello, I don't know English, I'm using a translator and I didn't even understand anything lol

thats okay, i told you the fix

**Comment by DevEriJMT** [comment]
What is quilt?

**Comment by GaucovyAero** [comment]
> What is quilt?

modified fabric

**Comment by DevEriJMT** [comment]
Axolot client is not available for Minecraft 1.18.2, the version which is the one I play because it works best for me ;(is Axolot mandatory to solve the problem?
